README.md

```diff
@@ -1,5 +1,11 @@
 # Velocidi iOS SDK in Objective-C 
```

Comment:
What do you think about having two rows for the badges?

![Cocoapods platforms](https://img.shields.io/cocoapods/p/VelocidiSDK.svg) ![Cocoapods](https://img.shields.io/cocoapods/v/VelocidiSDK.svg) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)

[![travis-ci](https://travis-ci.com/velocidi/velocidi-ios-objc-sdk.svg?branch=master)](https://travis-ci.com/velocidi/velocidi-ios-objc-sdk) [![codecov](https://codecov.io/gh/velocidi/velocidi-ios-objc-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/velocidi/velocidi-ios-objc-sdk)

P.S.: I took the idea from [Facebook](https://github.com/facebook/facebook-objc-sdk#facebook-sdk-for-ios).

Reply:
Personally prefer everything in just one line as it occupies less space vertically, but that's a personal preference and don't really have a strong opinion about it. @beatriz what do you think?

Reply:
I also don't have any strong opinion, but personally prefer the badges on one line.

Reply:
If you both prefer as it currently is, let's leave it.